edx_event_bus_kafka/internal/tests/test_config.py

```diff
@@ -50,13 +50,15 @@ def test_full(self):
                 EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS='localhost:54321',
                 EVENT_BUS_KAFKA_API_KEY='some_other_key',
                 EVENT_BUS_KAFKA_API_SECRET='some_other_secret',
+                EVENT_BUS_APP_NAME='my_client_id',
```

Comment:
Name?

Reply:
What do you mean?

Reply:
Originally, I thought this was supposed to be `EVENT_BUS_KAFKA_CLIENT_ID`.
However, an additional comment is that I think the following would make more sense:
```suggestion
                EVENT_BUS_APP_NAME='my_app_name',
```